src/page-object/moonstone/menu.ts

```diff
@@ -1,8 +1,10 @@
 import {BaseComponent} from '../baseComponent';
 import {getComponentByRole} from '../utils';
+import Chainable = Cypress.Chainable;
 
 export class Menu extends BaseComponent {
     static defaultSelector = '.moonstone-menu:not(.moonstone-hidden)'
+    static overlaySelector = '*[class^="moonstone-menu_overlay"]';
```

Comment:
curious about the selector, why did we went with a css selector instead of the normal dot notation, e.g. why not `.moonstone-menu_overlay` ? also, what’s the diff with having `*` in front vs none? are they not same behaviour?